nucypher/cli/commands/ursula.py

```diff
@@ -395,36 +395,36 @@ def forget(general_config, config_options, config_file):
 @option_force
 @group_general_config
 @click.option('--interactive', '-I', help="Run interactively", is_flag=True, default=False)
-@click.option('--prometheus', help="Run the ursula prometheus exporter", is_flag=True, default=False)
-@click.option('--metrics-port', help="Run a Prometheus metrics exporter on specified HTTP port", type=NETWORK_PORT)
-@click.option("--metrics-listen-address", help="Run a prometheus metrics exporter on specified IP address", default='')
-@click.option("--metrics-prefix", help="Create metrics params with specified prefix", default="ursula")
-@click.option("--metrics-interval", help="The frequency of metrics collection", type=click.INT, default=90)
+# @click.option('--prometheus', help="Run the ursula prometheus exporter", is_flag=True, default=False)
```

Comment:
What do you think about restoring this option and displaying a message into the console explaining the situation?  This might proactively reduce confusion.

Reply:
✔️  Fair enough. Updated